src/mixxx.cpp

```diff
@@ -116,6 +116,14 @@ MixxxMainWindow::MixxxMainWindow(QApplication* pApp, const CmdlineArgs& args)
     m_pShoutcastManager = NULL;
 #endif
 
+    //Reset pointers to cos
```

Comment:
Please use C++ initializations list. Feel free to clean up some other initializations as well.